### DISCUSSION TITLE
Button to see references in query results/on form

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5724] [Feature Requests]
**Requested By:** Dan at CSIRO

**Comment by maxpatiiuk** [comment]
Does the delete button work for this?
Or should we add another button to form (next to delete), that would essentially do the same thing - open delete blockers
One case where the delete button won't work is when the user has no delete permission to that table - if that's the issue, could add a separate button to the Form Meta menu
